src/mots/templates/directory.template.rst

```diff
@@ -1,14 +1,17 @@
 {# This Source Code Form is subject to the terms of the Mozilla Public #}
 {# License, v. 2.0. If a copy of the MPL was not distributed with this #}
-{# file, You can obtain one at https://mozilla.org/MPL/2.0/. #}
+{# file, You can obtain one at https://mozilla.org/MPL/2.0/.           #}
+..
+    This file was automatically generated using `mots export`.
```

Comment:
Where is "This file was automatically generated using `mots export`." coming from?

Reply:
This is literally part of the template, and will show up as a comment in anything generated by this template (e.g., `mots.rst`).

Reply:
Oh I see, I saw a `.rst` suffix and assumed this was part of the generated output. I would have expected `directory.rst.tmpl` or something as the filename. Either way it's fine.

Reply:
Putting `.rst` at the end tricks some editors and viewers/parsers to treat this template as reStructuredText, thus making it easier to edit and review, etc...